harness/helpers/events.ts

```diff
@@ -88,6 +88,10 @@ export const expectAggregateEvaluationEvent = ({
         metadata.configLastModified = lastModified
         metadata.clientUUID = expect.any(String)
     }
+    if (hasCapability(sdkName, Capabilities.evalReasons)) {
+        // Eval reasoning is specific to each test and is outside the scope of testing in general for these events.
+        metadata.eval = expect.any(Object)
```

Comment:
should the tests check for:
```suggestion
        metadata.evalReason = expect.any(Object)
```

Reply:
Eval is the spec'd parent object you have